lib/dry/core/memoizable.rb

```diff
@@ -56,6 +63,9 @@ class Memoizer < ::Module
           frozen: ::Kernel.instance_method(:frozen?)
         }.freeze
 
+        # @api private
+        attr_reader :methods
+
```

Comment:
@flash-gordon What was the reason for adding this ivar and reader? Scanning through the changes here, I can't see where it's used (I could likely be missing something obvious, though!)

Reply:
It's a leftover, initially, I was going to iterate over cached methods to re-memoize them but then I realized there's no need. Good catch!